symbol-processing-cmdline/build.gradle.kts

```diff
@@ -0,0 +1,59 @@
+import com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar
+
+val kotlinBaseVersion: String by project
+val signingKey: String? by project
+val signingPassword: String? by project
+
+plugins {
+    kotlin("jvm")
+    id("com.github.johnrengelman.shadow") version "6.0.0"
+    `maven-publish`
+    signing
+}
+
+val packedJars by configurations.creating
+
+dependencies {
+    packedJars(project(":compiler-plugin")) { isTransitive = false }
+    packedJars(project(":util")) { isTransitive = false }
+    packedJars(project(":api")) { isTransitive = false }
```

Comment:
Although it is more convenient for command line, I'm not sure if `:api` should be included here. When using K/N in Gradle, the classes of the API jar will appear twice in the classpath: one here and one from Gradle deps. Can you remove `:api` so as to keep the current behavior?